src/main/scala/org/camunda/feel/impl/parser/FeelParser.scala

```diff
@@ -68,7 +68,8 @@ object FeelParser extends JavaTokenParsers {
       "day of year",
       "day of week",
       "month of year",
-      "week of year"
+      "week of year",
+      "put all"
```

Comment:
Although the comment on this says it only requires function names with whitespaces. I wonder whether you also need to add `"put"` somewhere in the parsers. For example, `date` is also explicitly defined in this class, `time` and `not` as well, but `decimal` is not.

Reply:
The function name `put` can be parsed by the `name` parser. We don't need to handle it explicitly. It is different for names with whitespace because they aren't allowed as function names.